website/__init__.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env/ python
+
+from flask import Flask, session
+from .interfacing import Interfacer
+from .managers import LoginManager, UserManager, GameManager, TagManager
+from .generateFakeData import generateData
+
+
+config = { 
+        'database' : 'ZoomFront' 
+}
+# Create the Flask application and set its secret key.
+interfacer = Interfacer(config)
+loginManager = LoginManager(interfacer)
+tagManager = TagManager(interfacer)
+userManager = UserManager(interfacer)
+gameManager = GameManager(interfacer)
```

Comment:
No singletons! Instances parented to modules like this create _very_ nonobvious code paths!

Flask provides a right way to do this style of bringup and teardown through the `g` object: https://flask.palletsprojects.com/en/2.3.x/appcontext/#storing-data